packages/@aws-cdk/aws-amplify/test/app.test.ts

```diff
@@ -86,7 +86,7 @@ test('create an app connected to a GitLab repository', () => {
   // THEN
   expect(stack).toHaveResource('AWS::Amplify::App', {
     Name: 'App',
-    BuildSpec: '{\n  \"version\": \"1.0\",\n  \"frontend\": {\n    \"phases\": {\n      \"build\": {\n        \"commands\": [\n          \"npm run build\"\n        ]\n      }\n    }\n  }\n}',
+    BuildSpec: 'version: \"1.0\"\nfrontend:\n  phases:\n    build:\n      commands:\n        - npm run build\n',
```

Comment:
How about we just change one of these tests here, and leave the other ones as-is?

Reply:
sure, reverted and kept only one test with the new function